backend/development.ini.sample

```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
 email.notification.upload_permission_to_emitter.subject = [{website_title}] you give upload permission on {workspace_name} to {nb_receivers} users
```

Comment:
replace `you give upload permission on {workspace_name} to {nb_receivers} users` by `You allowed {nb_receivers} users to upload files on "{workspace_name}"`

Reply:
done, french translation ?

Reply:
`Vous avez autorisé {nb_receivers} utilisateurs à téléverser des fichiers dans "{workspace_name}"`